Bio/Phylo/_io.py

```diff
@@ -34,7 +34,7 @@
 
 
 def parse(file, format, **kwargs):
-    """Parse a file and return each of the trees it contains.
+    """Parse iteratively a file and return each of the trees it contains.
```

Comment:
How about:

```python
"""Parse a file iteratively, and yield each of the trees it contains.
```

In addition the the rephrasing, I suggest using yield rather than return given the Python specific meaning of these verbs.